src/components/GuestGallery.astro

```diff
@@ -28,7 +29,7 @@ const reel = [
 <div class="reel">
   {
     reel.map((image) => (
-      <img src={image.src.src} alt={image.alt} title="Coming soon" />
+      <Image alt={image.alt} src={image.src} title="Coming soon" />
```

Comment:
note (performance): we take of CLS by using Astro's Image component as it infers sizes automatically.